src/components/api-type-test.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from "express";
+import { components } from "../types/api";
+
+const list_results: components["schemas"]["Result"][] = [];
```

Comment:
This is a nice test of the API. Can you create a file `src/types/index.ts` that exports these subtypes on one level?

The syntax might be a bit different, but something like

```
export type Result = components["schemas"]["Result"]
```

So that you can use this simplified import here:

```suggestion
const results: Result[] = [];
```

Reply:
Sure